bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml

```diff
@@ -66,3 +66,21 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: NBDBLongElectionTimer
+      annotations:
+        message: |
+          OVN NB DB has a high election timer interval.
+      expr: |
+        ovn_db_cluster_election_timer{db_name="OVN_Northbound"} > 16000
```

Comment:
This is to check the existing limit and warn if this increases in future? If that's the case, that we might have to put this value a bit higher, because these values are based on 100 nodes with full workload. It might change when we move to 500 nodes (not baremetal) with full workload.

But if the idea is to warn that the value must not be set beyond x value for production environment, than i think we need to change this value to 3 seconds. Although it will generate the warning, but that's in my opinion is correct warning anyways.

Reply:
@vishnoianil The raft election is still being run only between the three masters, or is the number of masters also a function of the number of nodes in the cluster?

Reply:
@palanik1 Raft election is still being run between the three instances of NB or SB database. Election timer is affected by  how much load (db transaction and data replication) existing implementation can handle. WIth increase of the load, you may have to bump up the election timer.

Reply:
Right...it grows as a function of the number of the flows..which I guess can be approximated as a function of the number of nodes. Will make the threshold 3 sec for now.